.github/workflows/ci_cd.yml

```diff
@@ -107,6 +107,16 @@ jobs:
       - name: "Test with tox"
         run: tox -e tests -- -m "not integration"
 
+#      - name: "Upload coverage to Codecov"
```

Comment:
I originally wanted to add coverage checking, but I realized we don't have a token for this repo.

Reply:
I've added a secret if we want to use it. I suspect we'll be switching to an organization token at some point.